<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QWidget" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_Tables">
    <widget class="QStackedWidget" name="stackedWidget_Tables">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>531</width>
       <height>401</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_Schedule">
      <property name="maximumSize">
       <size>
        <width>531</width>
        <height>401</height>
       </size>
      </property>
      <widget class="QTableView" name="tableView_Schedule">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>290</y>
         <width>481</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>471</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_Employee">
      <widget class="QPushButton" name="pushButton_toAddEmployee">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>310</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить сотрудника</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_Employee">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>531</width>
         <height>311</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <column>
        <property name="text">
         <string>Ф.И.О.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Должность</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Телефон</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Зарплата $</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дата найма</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton_toEditEmployee">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>310</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_TeamList">
      <widget class="QTableView" name="tableView_TeamList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>461</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>401</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Расписание</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Штат сотрудников</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Команды</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_AddEmployee">
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <widget class="QGroupBox" name="groupBox_PasspotData">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Паспортные данные</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_Surname">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Surname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Фамилия:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Name">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Имя:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_SecondName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Отчество:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_SecondName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Birthday">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата рождения:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_itn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ИНН:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_itn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>0000000000</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_Birthday">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_PersonalData">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>371</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Данные сотрудника</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_Phone">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>+00(000)000-00-00</string>
      </property>
      <property name="text">
       <string>+()--</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Phone">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Телефон:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Position">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Должность:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Post">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_Salary">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Зарплата:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Salary">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>000000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Currency">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>$</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Recruitment">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Принят на работу:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_Recruitment">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_addEmployee">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>380</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_backFromAddEmployee">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>440</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_editEmployeeTitTitle">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>521</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавление нового сотрудника</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
